### ISSUE TITLE
[Dotcom shell]: QA: Multiple Links and buttons in the page which are in enabled state appears as disabled in high contrast themes

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9389]
### Description

Page Element:
Multiple links and buttons

Issue Description:
There are multiple lnks & buttons on page that are in enabled state but appear as disabled on applying high contrast black & white.

Expected:
Ensure that components appear in defined colors.

User Impact:
When the high contrast standards are not followed, high contrast users will face difficulty in accessing the application.
![image](https://user-images.githubusercontent.com/108443049/190962219-0263127b-600f-4727-960f-399b77c8f580.png)
![image](https://user-images.githubusercontent.com/108443049/190962228-bf31723a-4bb0-4db1-ac7e-514bebb1be4e.png)
![image](https://user-images.githubusercontent.com/108443049/190962251-28abae5d-1ea8-4f62-850f-aab0f7150f67.png)
![image](https://user-images.githubusercontent.com/108443049/190962274-dd8eec05-5ea4-4fc7-ac50-b73a019a7b17.png)
![image](https://user-images.githubusercontent.com/108443049/190962293-8c8ba400-5c66-4ee6-a34c-bb8d47abbcc1.png)
![image](https://user-images.githubusercontent.com/108443049/190962307-0bb3e4f6-224d-460f-a8ad-75bb4d681286.png)
![image](https://user-images.githubusercontent.com/108443049/190962318-4b43245f-631e-4285-ad08-73bd7d84a8eb.png)
![image](https://user-images.githubusercontent.com/108443049/190962337-54630663-c99e-4299-8347-4ee37d3bb54e.png)

### Component(s) impacted

Dotcom shell

### Browser

Firefox

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/iframe.html?id=components-dotcom-shell--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
Investigate if visited states are being added on top of high contrast mode
